llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -10,119 +10,155 @@
 // RISC-V subtarget features and instruction predicates.
 //===----------------------------------------------------------------------===//
 
+// Subclass of SubtargetFeature to be used when the feature is also a RISC-V
+// extension. Extensions have a version and may be experimental.
+//
+// name      - Name of the extension in lower case.q
+// major     - Major version of extension
+// minor     - Minor version of extension
+// fieldname - name of field to create in RISCVSubtarget.
+// desc      - Description of extension
+// implies   - Extensions or features implied by this extension.
+// value     - Value to assign to the field in RISCVSubtarget when this
+//             extension is enabled. Usually "true".
+class RISCVExtension<string name, int major, int minor, string fieldname,
+                     string desc, list<SubtargetFeature> implies = [],
+                     string value = "true">
```

Comment:
Are there any exceptions that `value` is not `true`? Or is it possible?

Reply:
Yes. Zvl*b is stored in a single unsigned value.